azkaban-web-server/src/main/java/azkaban/webapp/ExecutionLogsCleaner.java

```diff
@@ -25,10 +25,14 @@
    private static final long DEFAULT_EXECUTION_LOGS_RETENTION_MS = 3 * 4 * 7
        * 24 * 60 * 60 * 1000L;
 
-   // 1 hour
-   private static final long DEFAULT_LOG_CLEANUP_INTERVAL_SECONDS = 60 * 60;
+   // 1 minute
+   private static final long DEFAULT_LOG_CLEANUP_INTERVAL_SECONDS = 60;
```

Comment:
cleaning up every minute for something which has retention of 12 weeks seems too frequent. Do we know how long does it take on avg to execute this statement for 1000 deletions, and do we have over 1000 executions for clusters in a minute. 
This is not cleaning up everything too, we could probably run this clean up every hour with some batch_size in a loop while there are records to delete and committing each batch.

Reply:
48K log entries seems too less for clusters like UMP where at any given time 500-1500 flows are running with each flow running multiple jobs. However, I agree 1 minute is too aggressive. Lets find out how many logs are added in 24 hrs in UMP and then work on the frequency backwards.

Reply:
I will keep 1 hour as default. With that, we can set this differently based on load on cluster by setting config value for this frequency.